src/api/app/controllers/webui/request_controller.rb

```diff
@@ -75,8 +75,6 @@ def modify_review
   def show
     diff_limit = params[:full_diff] ? 0 : nil
     @req = @bs_request.webui_infos(filelimit: diff_limit, tarlimit: diff_limit, diff_to_superseded: @diff_to_superseded)
```

Comment:
you should also get rid of `webui_infos` as it is just returning a hash with the information you are getting rid of

Reply:
It won't happen in this PR, we want to avoid to change everything now, to reduce the numbers of merge conflict. We are going to submit a more aggressive second PR later

Reply:
I think this is part of this PR. You are leaving some unused code in here and the reason why there are that many variables is this method. So IMO this belongs together.

Reply:
`@req` is still used. For example to get the webui_actions hash. And that's another beast I would prefer to tackle in a separate PR to avoid merge conflicts (by other PRs) with changes we already did here.

Reply:
@Ana06 where are we leaving "unused code"?

Reply:
I am speaking about all the variables that return `webui_infos` that are not needed and with this PR also not used. In my opinion the way to go is getting rid of `webui_infos` and move the code here to the variables assignation. All this unneeded variables are there because of this method.

This refactoring (including gettting drop of `webui_infos`) is really simple, so IMO there is no reason to split it in a different PR.

Reply:
Ah, I misunderstood. There are some variables in [webui_infos](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request.rb#L888) that we might not use anymore and can be dropped. We will have a look.

Good catch:smiley: